docs/cmd/kn_broker_create.md

```diff
@@ -16,13 +16,26 @@ kn broker create NAME
   # Create a broker 'mybroker' in the 'myproject' namespace and with a broker class of 'Kafka'
   kn broker create mybroker --namespace myproject --class Kafka
 
+  # Create a broker 'mybroker' in the myproject namespace with config referencing configmap named spec-cm
+  kn broker create mybroker --namespace myproject --class Kafka --broker-config cm:spec-cm
+
+  # Create a broker 'mybroker' in the myproject namespace with config referencing configmap named spec-cm in test namespace
+  kn broker create mybroker --namespace myproject --class Kafka --broker-config cm:broker-spec-cm:test
+
+  # Create a broker 'mybroker' in the myproject namespace with config referencing configmap named spec-cm in test namespace
+  kn broker create mybroker --namespace myproject --class Kafka --broker-config cm:broker-spec-cm:test
+
```

Comment:
The last example seems to duplicate the example before. As @dsimansk mentioned, having an example for a secret and/or a full GVK would be helpful here.

Reply:
Thanks for the suggestion. I updated the examples accordingly. Hope they look okay now